### PR TITLE
Added the logic for Japannese search pages for description limiting t…

### LINKED ISSUE #537: Japanese search > search excerpt/snippet too long
image
https://main--sunstar--sunstar-global.hlx.page/jp/search?s=%E5%BE%93%E4%BA%8B%E8%80%85

@sdmcraft

### PR DESCRIPTION
**Pull request by meejain**
…o 1 line

Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after), along with a short summary of changes:

#537 

Fixes #537 

## Changelog:
_Please enter each change as a new bullet point_

## Test URLs:
- Original: https://www.sunstar.com/jp/search?s=%E5%BE%93%E4%BA%8B%E8%80%85
- Before: https://main--sunstar--hlxsites.hlx.page/jp/search?s=%E5%BE%93%E4%BA%8B%E8%80%85
- After: https://issue537-search-jp-pages--sunstar--hlxsites.hlx.page/jp/search?s=%E5%BE%93%E4%BA%8B%E8%80%85

## Library

- [ ] New Blocks introduced in this PR
      If yes, please provide details below
**Block Name** : For e.g. _cards_
- [ ] Documented in sidekick library

- [ ] New variations introduced in this PR
**Variation Name** :  For e.g. _cards (grid)_
- [ ] Documented in sidekick library

- [ ] New mixins introduced in this PR
**Mixin Name** :  For e.g. _compact, white_
- [ ] Documented in sidekick library

### CODE REVIEW

File: blocks/search-results/search-results.js [hunk 1]
```diff
@@ -132,7 +132,9 @@ async function searchPages(placeholders, term, page) {
     header.appendChild(link);
     res.appendChild(header);
     const para = document.createElement('p');
-    setResultValue(para, line.description, term);
+    if (getLanguage() === 'jp') {
```
Comment (jindaliiita) [inline blocks/search-results/search-results.js:135]: LGTM if @ehrro is fine with this logic.

**Review by jindaliiita — APPROVED** [review]

**Comment by ehrro** [comment]
Hello it looks good to me. Thanks.

File: blocks/search-results/search-results.js [hunk 2]
```diff
@@ -132,7 +132,9 @@ async function searchPages(placeholders, term, page) {
...
+      setResultValue(para, line.description.split('。')[0], term);
+    } else setResultValue(para, line.description, term);
```
Comment (sdmcraft) [inline blocks/search-results/search-results.js:137]: Why just `jp` ? Can't the same problem happen in any language with long description?

Comment (dnbute) [reply blocks/search-results/search-results.js:137]: Only `jp` has `。` so there should be another `if` for the other languages that use `.`

Comment (dnbute) [reply blocks/search-results/search-results.js:137]: Also maybe add `ja` as well to be safe, I don't think we have them mixed anywhere but better safe than sorry

Comment (sdmcraft) [reply blocks/search-results/search-results.js:137]: I mean what if the `line.description.` is long enough to require trimming.

**Pull request merged**